typescript/test/util.ts

```diff
@@ -0,0 +1,103 @@
+import * as assert from 'assert';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+
+const BASELINE_EXTENSION = '.baseline';
+export function compareToBaseline(outpurDir: string, baselineDir: string) {
+  // put all baseline files into fileStack
+  let fileStack: string[] = [];
+  const dirStack: string[] = [];
+  putAllBaselineFiles(baselineDir, fileStack, dirStack);
+  // store every item (file or directory with full path in output dir and baseline dir) in stack, pop once a time
+  const protoItemStack: Item[] = [];
+  putItemToStack(protoItemStack, outpurDir, baselineDir);
+  while (protoItemStack.length !== 0) {
+    const item = protoItemStack[0];
+    protoItemStack.shift();
+    // if item is a file, compare it with baseline
+    if (fs.lstatSync(item.outputPath).isFile()) {
+      const identical = checkIdenticalFile(
+        item.outputPath,
+        item.baselinePath + BASELINE_EXTENSION
+      );
+      // if two files are identical or it's generated properly, filter it from the stack.
+      if (identical !== 2) {
```

Comment:
2 is a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)), please use meaningful constants.

Reply:
Right! Thanks! was wondering how should I deal with that.